src/nvim/api/private/helpers.h

```diff
@@ -8,14 +8,9 @@
 #include "nvim/memory.h"
 #include "nvim/lib/kvec.h"
 
+void _api_set_error(Error *err, ErrorType errType, const char *format, ...);
```

Comment:
What's the idea behind this `_api_set_error` function name? I get that in some codebases the underscore implies it's private, but does Neovim actually do this too?